uploadNeuroDB/minc_deletion.pl

```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/perl
+use strict;
+use warnings;
+use Carp;
+use Getopt::Tabular;
+use FileHandle;
+use File::Basename;
+use File::Temp qw/ tempdir /;
+use File::Path qw/ make_path /;
+use Data::Dumper;
+use FindBin;
+use Cwd qw/ abs_path /;
+
+# These are the NeuroDB modules to be used
+use lib "$FindBin::Bin";
+use NeuroDB::File;
+use NeuroDB::MRI;
+use NeuroDB::DBI;
+use NeuroDB::Notify;
+use NeuroDB::MRIProcessingUtility;
+
+my $profile   = '';      # this should never be set unless you are in a
+                             # stable production environment
+my $seriesuid = '';
+my $query     = '';
+my $selORdel  = '';
+my @opt_table = (
+                 ["Basic options","section"],
+                 ["-profile     ","string",1, \$profile,
+                  "name of config file in ../dicom-archive/.loris_mri"
+                 ],
+                 ["-seriesuid", "string", 1, \$seriesuid, "Only deletes this SeriesUID"
+                 ],
```

Comment:
@gluneau 
can you please add description of the confirm and check if seriesUID is not set?